### PR TITLE
made the necessary changes to od

### PR DESCRIPTION
**Pull request by prasanjit15**
Made the necessary changes and faced some issues while automatic merging. So, this is a new PR in continuation to my first PR [#598](https://github.com/appwrite/appwrite/pull/598)

### CODE REVIEW

**Comment by prasanjit15** [comment]
@spielers @eldadfux  I have faced some issues in auto merging of the PR and hence created this new PR in continuation to my first PR.

**Comment by spielers** [comment]
@prasanjit15 Please resolve the conflict 

best,
spielers

**Comment by prasanjit15** [comment]
@spielers  I think the conflict is because it cannot find "od" files mentioned which may get resolved when my previous PR gets accepted.

**Comment by eldadfux** [comment]
@prasanjit15 we must have all the changes in a single PR for the tests to pass, and for us to know the changes don't break the server.

**Comment by spielers** [comment]
> @spielers I think the conflict is because it cannot find "od" files mentioned which may get resolved when my previous PR gets accepted.

Hey, @prasanjit15 You can merge all this in a single PR, so we can test all these

best,
spielers

**Comment by prasanjit15** [comment]
@spielers will do it but kindly don't do anything to the previous PR as this one will again take 14 days to review and I might as well miss the hacktoberfest goodies.

**Comment by spielers** [comment]
> @spielers I will do it but kindly don't do anything to the previous PR as this one will again take 14 days to review and I might as well miss the hacktoberfest goodies.

Yeah don't worry dude we are here to help You 
Best,
spielers

**Comment by prasanjit15** [comment]
Sure, I will do it as soon as possible

**Comment by prasanjit15** [comment]
@spielers @eldadfux do check this.

File: app/config/locale/translations/od.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+return [
+    'settings.inspire' => '"ବୁଦ୍ଧିମାନ ହବାର କଲା ଏମିତି ଏକ କଲା ଯାହା ଜଣାଇଥାଏ କୋଉ ଜିନ୍ସ କୁ ଅଣଦେଖା କରିବା ଉଚିତ |"', // This is the line printed in the homepage and console 'view-source'
+    'settings.locale' => 'od',
+    'settings.direction' => 'ltr',
+
+    // Service - Users
+    'account.emails.team' => '%s ଟିମ',
+    'account.emails.verification.title' => 'ଏକାଉଣ୍ଟ ଭେରିଫିକେସନ',
+    'account.emails.verification.body' => 'en.email.auth.confirm.tpl',
```
Comment (spielers) [inline app/config/locale/translations/od.php]: replace this to `od.email.auth.confirm.tpl'

File: app/config/locale/translations/od.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    'account.emails.recovery.title' => 'ପାସୱାର୍ଡ ରିସେଟ',
+    'account.emails.recovery.body' => 'en.email.auth.recovery.tpl',
```
Comment (spielers) [inline app/config/locale/translations/od.php]: replace this to `od.email.auth.recovery.tpl'

File: app/config/locale/translations/od.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    'account.emails.invitation.title' => 'ଈଂଭିଟେସନ %s ଟିମ at %s',
+    'account.emails.invitation.body' => 'en.email.auth.invitation.tpl',
```
Comment (spielers) [inline app/config/locale/translations/od.php]: replace this to `od.email.auth.invitation.tpl'

**Review by spielers — CHANGES_REQUESTED** [review]

**Comment by spielers** [comment]
Hey @prasanjit15  please do the suggested changes

Best,
spielers

**Comment by prasanjit15** [comment]
Sure...i checked it twice but still missed it..sorry will do it

**Comment by spielers** [comment]
Hey, @prasanjit15  Both the PRs #598  and #673  are some right. could you clarify The purpose of this?.

Best,
spielers

**Comment by prasanjit15** [comment]
@spielers As I said yesterday... When I did the changes that you and @eldadfux asked me to. The previous PR didn't got updated. I tried deleting the changes and did again but still nothing got updated. So, I thought maybe if I deleted the previous forked repo and fork once again the PR might get updated but unfortunately it didn't. So I have to create this again. ( Never did this type of thing before, as this is my first so faced this problem ). That is the reason I requested you yesterday if you are ok if I update the new PR with the same old PR and not to do anything with the old PR as I didn't want to loose my hacktoberfest goodies because of the 14 day review period which will start again with this new PR.

**Comment by prasanjit15** [comment]
@spielers I dont think any changes is required further. Please check again I think the changes you have asked to do had already been done. Please check again. All are already in od.

image

**Comment by spielers** [comment]
Will check soon !

**Comment by prasanjit15** [comment]
Sure, Let me know if any further improvement has to be done. Thanks!

**Review by spielers — APPROVED** [review]
Hey, @prasanjit15 Thanks for contributions will get you merged !!!

Best,
Spielers

**Comment by prasanjit15** [comment]
Thanks

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Pull request merged**